### PR TITLE
Node.js app workflow example

### PR DESCRIPTION
**Pull request by marcinc**
Addresses #320

### CODE REVIEW

File: examples/node-app/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+Open the browser at `http://localhost:8080`. You should see `Hello World` displayed on the screen.
+
+
+*NOTE*: The command above will use current kubectl context, and will attempt to deploy your app to `default` namespace. Those can be adjusted with `--kubecontext=<context>` & `--namespace=<ns>` options added to the command above.
+
+*IMPORTANT*: If `--kubecontext` is pointing at remote Kubernetes cluster you need to make sure that you adjust `image` in docker-compose.yaml file so that it points at registry you control and are able to push to. Should that be a private registry then make sure that the remote Kubernetes cluster is able to [pull images from it](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+
+Once the app has been build, pushed and deployed via _Kev's_ Skaffold integration you may inspect that the Node app is running in your cluster:
```
Comment (ezodude) [inline examples/node-app/README.md]: `has been build` -> `has been built`

File: README.md [hunk 2]
```diff
@@ -155,6 +155,7 @@ Check our [Roadmap][roadmap] for upcoming planned integrations.
 - [How does Kev differ from Kompose?](docs/tutorials/how-kev-differs-from-kompose.md)
 - [Getting started with Kev](docs/tutorials/getting-started-with-kev.md)
 - [Iterate on the app with Kev and Skaffold](docs/tutorials/kev-dev-with-skaffold.md)
+- [Simple Node.js app workflow example](docs/tutorials/simple-nodejs-app-workflow.md)
```
Comment (ezodude) [inline README.md:158]: ? Maybe rename to: From development to deployment, a simple Node.js example.

File: README.md [hunk 3]
```diff
@@ -155,6 +155,7 @@ Check our [Roadmap][roadmap] for upcoming planned integrations.
 - [How does Kev differ from Kompose?](docs/tutorials/how-kev-differs-from-kompose.md)
 - [Getting started with Kev](docs/tutorials/getting-started-with-kev.md)
 - [Iterate on the app with Kev and Skaffold](docs/tutorials/kev-dev-with-skaffold.md)
```
Comment (ezodude) [inline README.md:157]: Again ... I realise this is not part of this PR ... but maybe a rename ??
`Iterate on the app with Kev and Skaffold` -> `Develop and auto deploy with Kev and Skaffold`

File: README.md [hunk 4]
```diff
@@ -155,6 +155,7 @@ Check our [Roadmap][roadmap] for upcoming planned integrations.
 - [How does Kev differ from Kompose?](docs/tutorials/how-kev-differs-from-kompose.md)
 - [Getting started with Kev](docs/tutorials/getting-started-with-kev.md)
```
Comment (ezodude) [inline README.md:156]: I realise this is not part of this PR ... but maybe a rename ??
`Getting started with Kev` -> `Iterating with Kev`

File: docs/tutorials/simple-nodejs-app-workflow.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+Open the browser at `http://localhost:8080`. You should see `Hello World` displayed on the screen.
+
+
+*NOTE*: The command above will use current kubectl context, and will attempt to deploy your app to `default` namespace. Those can be adjusted with `--kubecontext=<context>` & `--namespace=<ns>` options added to the command above.
+
+*IMPORTANT*: If `--kubecontext` is pointing at remote Kubernetes cluster you need to make sure that you adjust `image` in docker-compose.yaml file so that it points at registry you control and are able to push to. Should that be a private registry then make sure that the remote Kubernetes cluster is able to [pull images from it](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+
+Once the app has been build, pushed and deployed via _Kev's_ Skaffold integration you may inspect that the Node app is running in your cluster:
```
Comment (ezodude) [inline docs/tutorials/simple-nodejs-app-workflow.md:75]: `has been build` -> `has been built`

**Review by ezodude — COMMENTED** [review]
Great example - minor comments.

File: README.md (hunk 2 above)
Comment (marcinc) [reply README.md:158]: It's not really about development to deployment journey - this might be covered by CI/CD elements tutorial. This one is covering a development/validation workflow only.

File: README.md (hunk 3 above)
Comment (marcinc) [reply README.md:157]: it's not only about deployment, as Skaffold takes care of build, push too. I think it can stay as is.

Comment (marcinc) [reply README.md:157]: Could change `Interate` -> `Develop` if that's any better.

File: README.md (hunk 4 above)
Comment (marcinc) [reply README.md:156]: Getting started sounds more user friendly.

File: README.md (hunk 2 above)
Comment (marcinc) [reply README.md:158]: Could rename `..app workflow...` -> `...app development workflow...` if that's better?

File: README.md (hunk 3 above)
Comment (ezodude) [reply README.md:157]: Let's go with `Develop`.

File: README.md (hunk 2 above)
Comment (ezodude) [reply README.md:158]: Yup that makes sense ^

**Review by ezodude — APPROVED** [review]
🚀
